rs.py

```diff
@@ -44,4 +44,4 @@ def classify():
 
 if __name__ == '__main__':
     configure_service('resources/config/app_config.cfg')
```

Comment:
Assuming there is nothing sensitive in them we can include them in the repo and therefore the image directly.

> On Jul 2, 2015, at 11:35 AM, Daniel Forsyth <notifications@github.com> wrote:
> 
> In rs.py:
> 
> > @@ -44,4 +44,4 @@ def classify():
> >  
> >  if __name__ == '__main__':
> >      configure_service('resources/config/app_config.cfg')
> They are 9.8mb in total.
> 
> —
> Reply to this email directly or view it on GitHub.
>